<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemMonitor 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RamProgressBar {
    text-align: center;
}
#RamProgressBar::chunk {
    background-color: #3498db;
}
#CpuProgressBar {
    text-align: center;
}
#CpuProgressBar::chunk {
    background-color: #2ecc71;
}
#DiskProgressBar {
    text-align: center;
}
#DiskProgressBar::chunk {
   background-color: #9b59b6;
}
QProgressBar
{
border: solid grey;
border-radius: 15px;
color: black;
}
QProgressBar::chunk 
{
border-radius :15px;
}    
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Resources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Memory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="RamProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ram_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Cpu:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="CpuProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Disk:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="DiskProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>461</width>
          <height>31</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="diskinfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>System Informations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="host">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Hostname:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pltf">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Platform:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pltf_ver">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Platform-version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pltf_re">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Platform-release:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ram">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>RAM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Processor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cpu_cores">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>CPU-Cores:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Ip-address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mac">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Mac-address:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
